person.rb

```diff
@@ -1,15 +1,16 @@
 require_relative './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It would be better if we set our `rentals` to both getter and setter so that we can perform both read and write operation instead of only reading the rentals value. ***HINT***: `attr_accessor`